src/components/general/Schedule.vue

```diff
@@ -1,13 +1,51 @@
 <template>
-	<h2>Schedule</h2>
 	<router-link v-if="link" to="/schedule">View full schedule ></router-link>
+	<div class="sidebar">
+		<h4 class="word">Topics <img :class="displayTopics ? 'arrow' : 'arrow-flipped'" src="@/assets/misc/CloseArrow.svg" @click="displayTopics = !displayTopics"></h4>
+		<span v-if="displayTopics">
+			<div style="color: black;" v-for="i in allTopics" :key="i">
+				<input type="checkbox" :id="i" :value="i" v-model="checkedTopics" class="checkbox">
+				<label for="i" class="checkboxText">{{ i }}</label>
+				<br>
+			</div>
+		</span>
+		<hr class="sidebar-hr" />
+
+		<h4 class="word">Format <img :class="displayFormats ? 'arrow' : 'arrow-flipped'" src="@/assets/misc/CloseArrow.svg" alt="" @click="displayFormats = !displayFormats"></h4>
+		<span v-if="displayFormats">
+			<div style="color: black;" v-for="i in allFormats" :key="i">
+				<input type="checkbox" :id="i" :value="i" v-model="checkedFormats" class="checkbox">
+				<label for="i" class="checkboxText">{{ i }}</label>
+				<br>
+			</div>
+		</span>
+	</div>
```

Comment:
I know this PR isn't focusing on CSS, but the UI isn't mobile compatible. Are you planning on doing this soon?